src/main/java/com/jcabi/log/MulticolorLayout.java

```diff
@@ -201,6 +243,16 @@ public String format(final LoggingEvent event) {
     }
 
     /**
+     * Checks if coloring is enabled.
+     * @return True iff coloring is enabled.
+     */
+    private boolean isColoringEnabled() {
```

Comment:
@diorcety what's the deal here? I believe there's already a way to disable color codes in the logs.